src/controls.rs

```diff
@@ -275,7 +274,10 @@ impl OrbitControls {
     fn rotate_left(&mut self, angle: f32) {
         self.spherical_delta.theta -= angle;
     }
-
+    pub fn angle_me(&mut self, angle: f32) {
```

Comment:
I changed the angle_me name to rotate_object . 
I decided not to put the angle code into the update method, since I'll have to handle cases when angle is not passed down to the update function .